linera-chain/src/block.rs

```diff
@@ -63,6 +68,42 @@ pub struct ConfirmedBlock {
     executed_block: ExecutedBlock,
 }
 
+#[async_graphql::Object(cache_control(no_cache))]
+impl ConfirmedBlock {
+    #[graphql(derived(name = "executed_block"))]
+    async fn _executed_block(&self) -> ExecutedBlock {
+        self.executed_block.clone()
+    }
+
+    async fn status(&self) -> String {
+        "confirmed".to_string()
+    }
+}
+
+#[async_graphql::Object(cache_control(no_cache), name_type)]
+impl Hashed<ConfirmedBlock> {
+    #[graphql(derived(name = "hash"))]
+    async fn _hash(&self) -> CryptoHash {
+        self.hash()
+    }
+
+    #[graphql(derived(name = "value"))]
+    async fn _value(&self) -> ConfirmedBlock {
+        self.inner().clone()
+    }
+}
+
+impl TryFrom<CertificateValue> for ConfirmedBlock {
+    type Error = &'static str;
```

Comment:
Should this be a `ConversionError` now, as in https://github.com/linera-io/linera-protocol/pull/2951?

Reply:
This PR is part of the bigger stack and I haven't merged `main` in yet. Will do.

Reply:
Done in [Merge remote-tracking branch 'origin/main' into storage-read-returns-…](https://github.com/linera-io/linera-protocol/pull/2950/commits/a71918bf501e9ad9126919d182f3eecc5405216e) (although it seems to bring in two commits oO).